StreamChatUITestsAppUITests/MockServer/MessageResponses.swift

```diff
@@ -10,103 +10,146 @@ extension StreamMockServer {
     
     func configureMessagingEndpoints() {
         server[MockEndpoint.message] = { [weak self] request in
-            self?.messageCreation(request: request) ?? .badRequest(nil)
+            let channelId = try! XCTUnwrap(request.params[EndpointQuery.channelId])
+            return self?.messageCreation(request, channelId: channelId) ?? .badRequest(nil)
         }
         server[MockEndpoint.messageUpdate] = { [weak self] request in
-            self?.messageUpdate(request: request) ?? .badRequest(nil)
+            self?.messageUpdate(request) ?? .badRequest(nil)
         }
         server[MockEndpoint.replies] = { [weak self] request in
-            self?.mockMessageReplies(request: request) ?? .badRequest(nil)
+            let messageId = try! XCTUnwrap(request.params[EndpointQuery.messageId])
+            return self?.mockMessageReplies(messageId) ?? .badRequest(nil)
         }
     }
     
-    func mockDeletedMessage(_ message: [String: Any]) -> [String: Any] {
+    func mockDeletedMessage(_ message: [String: Any], user: [String: Any]) -> [String: Any] {
         var mockedMessage = message
         mockedMessage[MessagePayloadsCodingKeys.deletedAt.rawValue] = TestData.currentDate
         mockedMessage[MessagePayloadsCodingKeys.type.rawValue] = MessageType.deleted.rawValue
+        mockedMessage[MessagePayloadsCodingKeys.user.rawValue] = user
         return mockedMessage
     }
     
     func mockMessage(
         _ message: [String: Any],
+        channelId: String,
         messageId: String?,
         text: String?,
+        user: [String: Any],
```

Comment:
```suggestion
        users: [String: Any],
```

Reply:
This is actually about only one user, why do you think it's better to rename this arg?